example_new/FactorsMean.Female.csv

```diff
@@ -1,102 +1,102 @@
-86,236.7626143,42.51571586,38.19289203,2456.080528,1482.463468,1.6,45.95307899,147.8858456
-87,236.479404,42.44555744,38.20380322,2453.049,1480.742846,1.6,45.92980819,147.7781515
-88,236.210657,42.36054793,38.21564664,2450.136471,1478.950146,1.6,45.83451912,147.6704575
-89,235.9532074,42.2623938,38.22754509,2447.315573,1477.084554,1.6,45.78815882,147.5627635
-90,235.7053427,42.1534847,38.23882201,2444.574369,1475.158021,1.6,45.71234754,147.4550695
-91,235.4667492,42.03680987,38.24899709,2441.915782,1473.194274,1.6,45.65710673,147.3473755
-92,235.2383937,41.91584544,38.25777401,2439.356288,1471.22728,1.6,45.55394764,147.2396815
-93,235.022344,41.79441632,38.26502084,2436.923938,1469.299206,1.6,45.51883148,147.1319875
-94,234.8215352,41.67653817,38.27074405,2434.655769,1467.45796,1.6,45.45416583,147.0242935
-95,234.6394942,41.56624638,38.27505751,2432.594752,1465.754424,1.6,45.40678927,146.9165995
-96,234.480036,41.4674198,38.2781483,2430.78644,1464.239516,1.6,45.30332472,146.8089055
-97,234.346951,41.38360784,38.28024138,2429.275515,1462.96124,1.6,45.25050609,146.7012114
-98,234.2437024,41.31786958,38.2815654,2428.10246,1461.961897,1.6,45.17309058,146.5935174
-99,234.1731544,41.27263331,38.28232202,2427.300564,1461.275605,1.6,45.06665898,146.4858234
-100,234.1373493,41.24958386,38.28266066,2426.893479,1460.926295,1.6,45.05224962,146.3781294
+Age,FoodCarbohydrate,FoodFat,FoodProtein,FoodSodium,EnergyIntake,PhysicalActivity,Weight,Height
+0,221.1476524,31.13789942,33.92813043,2196.016303,1300.544226,1.6,3,51
+1,221.7277743,31.21901833,34.01566318,2201.709517,1303.944915,1.6,9,73
+2,222.8749365,31.37973972,34.18880425,2212.974562,1310.672621,1.6,11.5,86
+3,224.563453,31.61708048,34.44377373,2229.573015,1320.582631,1.6,14,94
+4,226.7559628,31.92668769,34.77507071,2251.157879,1333.464323,1.6,15,101
+5,229.4047651,32.30297815,35.17566658,2277.285734,1349.04853,1.6,16.5,107.1936339
+6,232.4534954,32.73931352,35.6372475,2307.431965,1367.016793,1.6,19,112.8924717
+7,235.8390579,33.22820312,36.15049453,2341.008312,1387.012038,1.6,21,117.9573555
+8,239.4937245,33.76152487,36.70538786,2377.381897,1408.650173,1.6,24,122.7415713
+9,243.3473014,34.3307548,37.29152143,2415.894838,1431.532085,1.6,27,127.5942064
+10,247.3292678,34.92719553,37.89841397,2455.883577,1455.255487,1.6,31,132.8497119
+11,251.3707956,35.54219486,38.51580352,2496.697127,1479.42615,1.6,35.5,138.5804571
+12,255.4065726,36.16734686,39.13391425,2537.713513,1503.668069,1.6,40,143.8308082
+13,259.3763634,36.79466917,39.74368617,2578.353879,1527.632221,1.6,43.5,147.6832203
+14,263.2262638,37.41675232,40.3369613,2618.093857,1551.003671,1.6,46.5,150.3832374
+15,266.9096205,38.02687836,40.90662243,2656.471986,1573.506877,1.6,48.5,152.3876156
+16,270.3876104,38.61910818,41.44668323,2693.09516,1594.909148,1.6,50,153.824056
+17,273.6294892,39.18833846,41.95233133,2727.64121,1615.022328,1.6,51,154.7357344
+18,335.8521047,48.23901059,51.50461471,3350.91255,1983.577973,1.6,52,155.1705973

```

Comment:
There seems to be quite a large step here from age 17 to 18. Is that intentional?